tests/org.eclipse.xtext.xbase.testlanguages/src/org/eclipse/xtext/xbase/testlanguages/GenerateXbaseTestLanguages.mwe2

```diff
@@ -1,11 +1,10 @@
-module org.eclipse.xtext.xbase.testlanguages.GenerateXImportSectionTestLang
+module org.eclipse.xtext.xbase.testlanguages.GenerateXbaseTestLanguages
 
 import org.eclipse.emf.mwe.utils.*
 import org.eclipse.xtext.generator.*
 import org.eclipse.xtext.ui.generator.*
 
-var grammarURI = "classpath:/org/eclipse/xtext/xbase/testlanguages/XImportSectionTestLang.xtext"
-var fileExtensions = "__ximportimporttestlang"
+var fileExtensions = ""
```

Comment:
I might need your help here then: what's the right way to do that (I was setting it to an empty string because I see you've already done that in another testlanguages project in the Xtext code base): If I do

```
    	language = auto-inject {
    		fileExtensions = "__ximportimporttestlang"
    		uri = "classpath:/org/eclipse/xtext/xbase/testlanguages/XImportSectionTestLang.xtext"
```

the generated standalone setup will have the method:

```
	@Override
	public List<String> getFileExtensions() {
		return Arrays.asList("__ximportimporttestlang");
	}
```

but the register method uses the wrong file extension

```
		org.eclipse.xtext.resource.IResourceServiceProvider.Registry.INSTANCE.getExtensionToFactoryMap().put("ximportsectiontestlang", serviceProvider);
```

which breaks the tests...